test/functional/tests/io_class/test_io_class_eviction_priority.py

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+from collections import namedtuple
+from math import isclose
+from datetime import timedelta
+import random
+
+import pytest
+
+from .io_class_common import *
+from api.cas.cache_config import CacheMode, CacheLineSize
+from api.cas.casadm_params import OutputFormat
+from api.cas.ioclass_config import IoClass
+from api.cas.statistics import UsageStats
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools import fs_utils
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from test_utils.filesystem.file import File
+from test_utils.os_utils import drop_caches, DropCachesMode, sync, Udev
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrize("cache_mode", [CacheMode.WT])
```

Comment:
I don't think it has to be passed as parametrize when there is only one value of this parameter